test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/flag/TestCompileCommandFileWriter.java

```diff
@@ -44,7 +44,7 @@
  * @build jdk.test.whitebox.WhiteBox
  * @run driver jdk.test.lib.helpers.ClassFileInstaller jdk.test.whitebox.WhiteBox
  * @run junit/othervm -Xbootclasspath/a:. -DSkipWhiteBoxInstall=true -XX:+IgnoreUnrecognizedVMOptions -XX:+UnlockDiagnosticVMOptions
- *                    -XX:+WhiteBoxAPI compiler.lib.ir_framework.flag.TestCompilePhaseCollector
+ *                    -XX:+WhiteBoxAPI compiler.lib.ir_framework.flag.TestCompileCommandFileWriter
```

Comment:
Do we really need the WhiteBoxAPI? `-XX:+IgnoreUnrecognizedVMOptions` can be removed as well. Please also update the copyright date.

Reply:
We do need the WhiteBoxAPI. IgnoreUnrecognizedVMOptions removed, date updated.